index.html

```diff
@@ -54,7 +54,192 @@ <h1 class="title">
+                </ul>
+                <button class="card-button">See Project</button>
+            </div>
+        </div>
+    </section>
+    <section id="aboutme">
+        <div class="content">
+            <h1 class="title">
+                About
+                <br>
+                Myself
+            </h1>
+            <p class="description">
+                Hello I'm a software developer! I can help you build a product , feature or website Look through some of
+                my work and
+                experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+            </p>
+            <p class="connect-link">
+                LET'S CONNECT
+            </p>
+            <ul class="social-media">
+                <li>
+                    <a href="https://github.com/Rimuru-Tempest-26" target="_blank"><img src="images/icons/github.svg"
+                            alt="Github icon"></a>
+                </li>
+                <li>
+                    <a href="https://www.linkedin.com/in/phone-khaing-hein-759497216/" target="_blank"><img
+                            src="images/icons/linkedin.svg" alt="Linkedin icon"></a>
+                </li>
+                <li>
+                    <img src="images/icons/peace.svg" alt="Peace icon">
+                </li>
+                <li>
+                    <a href="https://twitter.com/PhoneKhaingHein" target="_blank"><img src="images/icons/twitter.svg"
+                            alt="Twitter icon"></a>
+                </li>
+                <li>
+                    <a href="mailto:pkh2662003@gmail.com" target="_blank"><img src="images/icons/gmail.svg"
+                            alt="Gmail icon"></a>
+                </li>
+            </ul>
+            <button class="aboutme-button">Get my resume</button>
+        </div>
+        <div id="skills">
+            <ul>
+                <li class="skill-title">
+                    <div class="skill-list">
+                        Languages
+                        <img src="images/icons/down-arrow.svg" alt="Down arrow">
+                    </div>
+                    <div class="skill-detail">
+                        <img src="images/icons/javascript.svg" alt="Javascript image">
+                        <a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript">Javascript</a>
+                    </div>
+                    <div class="skill-detail">
+                        <img src="images/icons/html.svg" alt="HTML image">
+                        <a href="https://developer.mozilla.org/en-US/docs/Web/HTML">HTML</a>
+                    </div>
+                    <div class="skill-detail">
+                        <img src="images/icons/css.svg" alt="CSS image">
+                        <a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a>
+                    </div>
```

Comment:
A nested `ul`, `li`, `ul`, and `li` structure sounds dope, right? 🤯 mind-blowing but would make your project so much better, happens to also be one of the requirements:
- [ ] The technology in each category(e.g., "JavasScript", "HTML" and "CSS") should also be a list `ul` and each of them a li.

You could maybe make `Languages` the `ul` and then  "JavasScript", "HTML" and "CSS" would be the `li`s, thoughts? 🤔

Reply:
Fixed.